examples/README.md

```diff
@@ -108,6 +109,11 @@ normal React hooks to State management libraries. Check the full list with more
 - [MobX State Tree](./state-management/examples/bidirectional-mobx-state-tree/)
 - [React Easy State](./state-management/examples/bidirectional-easy-state/)
 
+## Node Package Managers
```

Comment:
Can we move this at the very beginning of the examples list? this analysis is more important than the rest 🥳

Reply:
Yes for sure! I already moved it.